presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkConfig.java

```diff
@@ -288,4 +295,95 @@ public PrestoSparkConfig setOutOfMemoryRetrySparkConfigs(String outOfMemoryRetry
         this.outOfMemoryRetrySparkConfigs = MAP_SPLITTER.split(nullToEmpty(outOfMemoryRetrySparkConfigs));
         return this;
     }
+
+    public DataSize getAverageInputDataSizePerExecutor()
+    {
+        return this.averageInputDataSizePerExecutor;
```

Comment:
remove `this`. Same for all other getters. We don't use `this` outside the constructors.